lib/screenplay_web/controllers/auth_controller.ex

```diff
@@ -23,7 +22,7 @@ defmodule ScreenplayWeb.AuthController do
       ScreenplayWeb.AuthManager,
       username,
       %{roles: roles},
-      ttl: {expiration - current_time, :seconds}
+      ttl: {session_ttl_hours, :hours}
```

Comment:
Does this change mean that every time this `callback` action gets hit (via `/:provider/callback`, looks like) the session TTL gets reset to 30 days? As opposed to the previous code, which appeared to have the session expire at the date/time specified by `auth.credentials.expires_at` no matter how many times this action ran.

(When does this action run? Is it only once per session, or every time a request that requires authorization comes in?)

Reply:
Gonna be honest that I'm still a little fuzzy on this. I believe this code only runs when no session exists at all. So basically the first time you log in to the app. It takes `ttl` amount of time for the session to expire. After it expires, you would need to refresh the session (can only be done via a page refresh atm, but could use a refresh token to handle this in the future) and run this action again. Bumping it out to 30 days means that we can run this action a lot less frequently because it takes much longer for the session to expire.